src/models.rs

```diff
@@ -44,13 +54,16 @@ impl DownloadRecordExt for DownloadRecord {
   }
 }
 
-#[derive(Debug, Clone, Default, PartialEq, Serialize, Deserialize)]
-#[serde(rename_all = "SCREAMING_SNAKE_CASE")]
-pub enum DownloadState {
-   #[default]
-   Unknown,
-   Created,
-   InProgress,
-   Paused,
-   Cancelled,
-}
+impl fmt::Display for DownloadState {
+  fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+      let text = match self {
+        DownloadState::Unknown => "Unknown",
+        DownloadState::Created => "Created",
+        DownloadState::InProgress => "InProgress",
+        DownloadState::Paused => "Paused",
+        DownloadState::Cancelled => "Cancelled",
+        DownloadState::Completed => "Completed",
+      };
+      write!(f, "{}", text)
+  }
+}
```

Comment:
I think the file needs a newline at the end?

Reply:
Fixed.